### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/interface/theme-color.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#499] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/interface/theme-color.html

主题允许你自定义主题颜色，甚至提供选择器。

https://theme-hope.vuejs.press/zh/guide/interface/theme-color.html

**Comment by bladedevoff** [comment]
真不戳
